From patchwork Sat Nov 12 13:46:33 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [LEDE-DEV, v2, 3/3] apm821xx: remove replaced netgear,
 wndr4700-usb dwc2 definiton
From: Christian Lamparter <chunkeey@googlemail.com>
X-Patchwork-Id: 694030
Message-Id: <7923048c19178ec68ed95d41e5e2892ef9297304.1478957900.git.chunkeey@gmail.com>
To: lede-dev@lists.infradead.org
Cc: Chris Blake <chrisrblake93@gmail.com>
Date: Sat, 12 Nov 2016 14:46:33 +0100

This special dwc2 device definition for the Netgear WNDR4700
has been replaced by amcc,usb-otg-405ex.

Signed-off-by: Christian Lamparter <chunkeey@gmail.com>
---
 .../800-usb-dwc2-add-wndr4700-otg.patch            | 48 ----------------------
 1 file changed, 48 deletions(-)
 delete mode 100644 target/linux/apm821xx/patches-4.4/800-usb-dwc2-add-wndr4700-otg.patch

diff --git a/target/linux/apm821xx/patches-4.4/800-usb-dwc2-add-wndr4700-otg.patch b/target/linux/apm821xx/patches-4.4/800-usb-dwc2-add-wndr4700-otg.patch
deleted file mode 100644
index eee3fd0..0000000
--- a/target/linux/apm821xx/patches-4.4/800-usb-dwc2-add-wndr4700-otg.patch
+++ /dev/null
@@ -1,48 +0,0 @@
---- a/drivers/usb/dwc2/platform.c
-+++ b/drivers/usb/dwc2/platform.c
-@@ -115,6 +115,37 @@ static const struct dwc2_core_params par
- 	.hibernation			= -1,
- };
- 
-+static const struct dwc2_core_params params_wndr4700 = {
-+	.otg_cap			= DWC2_CAP_PARAM_NO_HNP_SRP_CAPABLE,
-+	.otg_ver			= -1,
-+	.dma_enable			= -1,
-+	.dma_desc_enable		= -1,
-+	.speed				= -1,
-+	.enable_dynamic_fifo		= -1,
-+	.en_multiple_tx_fifo		= -1,
-+	.host_rx_fifo_size		= -1,
-+	.host_nperio_tx_fifo_size	= -1,
-+	.host_perio_tx_fifo_size	= -1,
-+	.max_transfer_size		= -1,
-+	.max_packet_count		= -1,
-+	.host_channels			= -1,
-+	.phy_type			= -1,
-+	.phy_utmi_width			= -1,
-+	.phy_ulpi_ddr			= -1,
-+	.phy_ulpi_ext_vbus		= -1,
-+	.i2c_enable			= -1,
-+	.ulpi_fs_ls			= -1,
-+	.host_support_fs_ls_low_power	= -1,
-+	.host_ls_low_power_phy_clk	= -1,
-+	.ts_dline			= -1,
-+	.reload_ctl			= -1,
-+	.ahbcfg				= GAHBCFG_HBSTLEN_INCR16 <<
-+					  GAHBCFG_HBSTLEN_SHIFT,
-+	.uframe_sched			= -1,
-+	.external_id_pin_ctl		= -1,
-+	.hibernation			= -1,
-+};
-+
- static int __dwc2_lowlevel_hw_enable(struct dwc2_hsotg *hsotg)
- {
- 	struct platform_device *pdev = to_platform_device(hsotg->dev);
-@@ -309,6 +340,7 @@ static int dwc2_driver_remove(struct pla
- static const struct of_device_id dwc2_of_match_table[] = {
- 	{ .compatible = "brcm,bcm2835-usb", .data = &params_bcm2835 },
- 	{ .compatible = "rockchip,rk3066-usb", .data = &params_rk3066 },
-+	{ .compatible = "netgear,wndr4700-usb", .data = &params_wndr4700 },
- 	{ .compatible = "snps,dwc2", .data = NULL },
- 	{ .compatible = "samsung,s3c6400-hsotg", .data = NULL},
- 	{},
